src/settypes/duplicatevector.jl

```diff
@@ -35,6 +31,6 @@ end
 Base.iterate(dv::DuplicateVector)             = iterate(collect(dv))
 Base.iterate(dv::DuplicateVector, i::Integer) = iterate(collect(dv), i)
 
-function ×(a::DuplicateVector{T}, b::DuplicateVector{T}) where {T}
+function product(a::DuplicateVector{T}, b::DuplicateVector{T}) where {T}
```

Comment:
Commenting here out of context: has #63 been fixes?

Reply:
Still open so no.